samples/dect/dect_phy/hello_dect/Kconfig

```
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config CARRIER
+	int "Carrier to use"
+	default 0
+	help
+	   The availability of the channels and the exact regulation to use them varies in different countries.
+	   See ETSI TS 103 636-2 5.4.2 for the calculation.
+
+config NETWORK_ID
+	int "Network ID"
+	range 1 4294967295
+	default 91
+
+config MCS
+	int "MCS"
+	default 1
+	help
+	   The MCS impacts how much data can be fit into each subslot.
```

```suggestion
	   The MCS impacts how much data can fit into each subslot.
```

or

```suggestion
	   The MCS impacts how much data can be fitted into each subslot.
```

I prefer the first one.